extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/IcebergCatalogRangerSparkExtensionSuite.scala

```diff
@@ -47,6 +47,8 @@ class IcebergCatalogRangerSparkExtensionSuite extends RangerSparkExtensionSuite
   val namespace1 = icebergNamespace
   val table1 = "table1"
   val outputTable1 = "outputTable1"
+  val bobNamespace= "default_bob"
+  val bobSelectTable = "table_select_bob_1"
```

Comment:
Query permission is required when using ReplaceData to execute `DELETE/UPDATE/MERGEINTO` in spark 3.5. So I created a new `default_bob.table_select_bob_1` table. For bob, it only has select permissions but no update permissions.